src/client/BUILD

```diff
@@ -1,3 +1,5 @@
+# BUILD file for ProtoCached client
+package(default_visibility = ["//visibility:public"])
```

Comment:
Nit again...Empty line after this line? Maybe we should just enable some sort of formatter, I feel this happens quite a lot.